rfcs/meta_yml_notify.md

```diff
@@ -0,0 +1,50 @@
+# RFC 37: Split META.yml `suggested_reviewers` into `suggested_reviewers` and `notify`
+
+## Summary
+
+Change `META.yml` files in wpt to have `notify` in addition to `suggested_reviewers`.
+@wpt-pr-bot mentions everyone in `notify` and then requests review from one person from `suggested_reviewers`.
+
+## Details
+
+At TPAC 2019 we set out some [priorities for 2020](https://docs.google.com/document/d/1gie7LFb6cAUfabY86MYuWM7m7ux_FaKhDkLdpz0zWkg/edit),
+which include increasing PR review velocity and reduce the number of stale PRs.
+
+wpt has META.yml files with `suggested_reviewers`. @wpt-pr-bot will request review from everyone listed in `suggested_reviewers` for PRs that change that directory, and also assign one of them in a round-robin fashion.
+
+See https://web-platform-tests.org/reviewing-tests/index.html#notifications
+
+The documentation only says that `suggested_reviewers` is for getting notifications for PRs.
+
+This has a few problems:
+
+* There is a disconnect between what contributors would reasonably expect and what often actually happens. When people are requested review and someone is assigned, it communicates that the person assigned will help the contributor to get it landed.
+* The current system doesn't support varying preferences from reviewers. Some people want to see all PRs for a given folder (which is supported by the current system), while others want to help with reviews but not be notified for everything, and some may want to be notified for everything but can't commit to helping with reviews.
+
+To address these problems, have two separate keys in `META.yml`:
+
+* `suggested_reviewers`: @wpt-pr-bot will only request review from one of these, similar to how assignee currently works.
+* `notify`: @wpt-pr-bot will add a comment (before requesting reviews) notifying everyone listed.
+
+Either @wpt-pr-bot can continue to also assign the selected reviewer, or it can only request review and not assign anyone. (This should be decided before merging this RFC.)
```

Comment:
I think it'd be better to stop using the assignee field, even though that means I'll have to update mail filters again.

Reply:
Done.